db/migrations/campaign/V4.0.0.4__table_campaigns.sql

```diff
@@ -0,0 +1,21 @@
+BEGIN TRANSACTION;
+
+CREATE TABLE IF NOT EXISTS campaign.beneficiaries (
+    id uuid NOT NULL DEFAULT (gen_random_uuid()),
+    name character varying(100) NOT NULL,
```

Comment:
We can split the `name` in `first_name`, `last_name` to ensure that we can use them separatelly.

Reply:
Done.